starknet_devnet/server.py

```diff
@@ -85,6 +84,26 @@ def set_gas_price(args):
     """Assign gas_price"""
     state.starknet_wrapper.set_gas_price(args.gas_price)
 
+#We don't need init method here.
+#pylint: disable=W0223
+class Devnet(BaseApplication):
+    """Our Gunicorn application."""
+
+    def __init__(self, application, args):
+        self.args = args
+        self.application = application
+        super().__init__()
+
+    def load_config(self):
+        self.cfg.set("bind", f"{self.args.host}:{self.args.port}")
+        self.cfg.set("workers", 1)
```

Comment:
How does it handle simultaneous requests?

Reply:
It queues the requests just like meinheld did and servers them.
I run 7 simultaneous requests for get_balance just to make sure.

**meinheld** version:
![image](https://user-images.githubusercontent.com/10278469/182779223-991cf4db-84c6-4ea6-8afc-ed66b595e67f.png)

**gunicorn** version:
![image](https://user-images.githubusercontent.com/10278469/182779238-7fc02689-a186-4be7-a2ba-20121904da63.png)

Code I used:
```
x = () => fetch("http://127.0.0.1:5050/account_balance?address=ACCOUNT"
Promise.all([x(), x(), x(), x(), x(), x(), x()])
```

We can't have more than one worker as the state between them is not shared.

Reply:
Great, thanks for the explanation and visualization!

Reply:
Calls to contracts could potentially be run simultaneously, but probably not with flask, it starts an event loop per request. Do users complain about it?

Reply:
I remember something like that being mentioned some half a year ago.

Reply:
It is should be possible to optimise calls by executing them asynchronously on a CPU bound executor or by forking the process, but this is a discussion for [other](https://github.com/Shard-Labs/starknet-devnet/issues/208) probably.